src/generated-client.js

```diff
@@ -224,6 +227,7 @@ class GeneratedApiClient {
     ];
 
     const request = this.http.postJson(url, {
+      headers: { 'Content-Type': 'application/x-x509-ca-cert' },
```

Comment:
`application/x-x509-ca-cert` is a string, `http.postJson` is going to try JSON stringify that and fail?

Reply:
looks like the http.json will only try to set
```
{
  'Content-Type': 'application/json',
  'Accept': 'application/json'
}
```
as default header, but override it by using provided headers from input. No JSON.stringify logic on body. 
But yes, this change is confusing... I'll make changes.